install-deps.sh

```diff
@@ -343,11 +343,11 @@ else
 	if [ "$control" != "debian/control" ] ; then rm $control; fi
         ;;
     centos|fedora|rhel|ol|virtuozzo)
-        yumdnf="yum"
-        builddepcmd="yum-builddep -y --setopt=*.skip_if_unavailable=true"
-        if test "$(echo "$VERSION_ID >= 22" | bc)" -ne 0; then
-            yumdnf="dnf"
-            builddepcmd="dnf -y builddep --allowerasing"
+        yumdnf="dnf"
+        builddepcmd="dnf -y builddep --allowerasing"
+        if ( [ "$ID" = "centos" ] || [ "$ID" = "rhel" ] ) && version_lt $VERSION_ID 8; then
```

Comment:
```suggestion
        if [[ $ID =~ centos|rhel ]] && version_lt $VERSION_ID 8; then
```

if `[[ expression ]]` is allowed.

Reply:
Nice one @tchaikov